extensions/vscode/webviews/homeView/src/stores/home.ts

```diff
@@ -85,12 +89,18 @@ export const useHomeStore = defineStore("home", () => {
    * @param name the name of the new configuration to select
    * @returns true if the selected contentRecord was the same, false if not
    */
-  function updateSelectedConfigurationByName(name: string) {
+  function updateSelectedConfigurationByName(
+    name: string,
+    projectDir?: string,
```

Comment:
I'm wondering if this should be optional to avoid us changing a selected configuration incorrectly.

Reply:
should be or shouldn't be? I'm thinking shouldn't be, for the reason you mention?

Reply:
Yes sorry, _shouldn't_ be optional